src/lib.rs

```diff
@@ -14,7 +14,7 @@
 
 //! Tremor runtime
 
-#![deny(warnings)]
+// #![deny(warnings)]
```

Comment:
:scream_cat:

Reply:
Not here directly but worth the contest, there have been discussions on removing `deny(warnings)` from code and instead of moving it to CI only to make refactoring / testing easier. That might be something we should consider

Reply:
I removed this lint because I wanted to quietly start working. I do think it's a super good idea to guarantee no warning (from both `rustc` and `clippy`) on `main`, but it makes things hard when working with unfinished code.

Anyway, I promise i'll bring the lints back to errors once everything is implemented the way it should!

Reply:
I'd say lets disable it here and enable it for CI.

Reply:
I'll make a ticket

Reply:
we already had a ticket :D https://github.com/tremor-rs/tremor-runtime/issues/1353 thanks @marioortizmanero  for bringing this up and ticketing it in the past!